puma_motor_driver/src/multi_controller_node.cpp

```diff
@@ -157,12 +162,27 @@ int main(int argc, char *argv[])
   ros::NodeHandle nh_private("~");
 
   std::string serial_port;
-  nh_private.param<std::string>("port", serial_port, "/dev/ttyUSB0");
+  std::string canbus_dev;
 
-  serial::Serial serial;
-  serial.setPort(serial_port);
-  puma_motor_driver::SerialGateway gateway(serial);
+  boost::scoped_ptr<puma_motor_driver::Gateway> gateway;
 
-  MultiControllerNode n(nh, nh_private, gateway);
+  if (nh_private.getParam("canbus_dev", canbus_dev))
+  {
+    gateway.reset(new puma_motor_driver::SocketCANGateway (canbus_dev));
+  }
+  else if (nh_private.getParam("port", serial_port))
```

Comment:
Let's change this param to be serial_port, to clarify the relationship between serial and CAN.